.travis.yml

```diff
@@ -37,7 +47,7 @@ install:
   - ./build/mvn --version
 
 script:
-  - ./build/mvn clean install -Dmaven.javadoc.skip=true -V
+  - ./build/mvn clean install -Dmaven.javadoc.skip=true -V -B -ntp -Dorg.slf4j.simpleLogger.defaultLogLevel=warn
```

Comment:
How about test compile on ARM64 first? Then we can enable test moudle by moudle after fixing?
```suggestion
  - export MVN_ARGS="-Dmaven.javadoc.skip=true -V -B -ntp -Dorg.slf4j.simpleLogger.defaultLogLevel=warn"
  - ./build/mvn clean install -DskipTests $MVN_ARGS
# - ./build/mvn test $MVN_ARGS
```